### PR TITLE
add shipit handlers

### PR DESCRIPTION
**Pull request by dwradcliffe**
WIP, still need to add templates and test

@maclover7

### CODE REVIEW

File: lib/lita/handlers/deploy_notifications.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        seconds = seconds % 60
+
+        if minutes > 0
+          "#{minutes}m#{seconds.to_s.rjust(2, '0')}s"
+        else
+          "#{seconds}s"
+        end
+      end
+    end
+    Lita.register_handler(x)
```
Comment (maclover7) [inline lib/lita/handlers/deploy_notifications.rb]: Doesn't it need to register the DeployNotifications handler here?

Comment (dwradcliffe) [inline lib/lita/handlers/deploy_notifications.rb]: Yeah, typo

File: lita_config.rb [hunk 2]
```diff
@@ -40,6 +44,8 @@
   ## documentation for options.
   # config.handlers.some_handler.some_config_key = "value"
 
+  config.handlers.shipit.token = ENV["SHIPIT_TOKEN"]
```
Comment (maclover7) [inline lita_config.rb:49]: Can you document this in the README?

**Comment by dwradcliffe** [comment]
@maclover7 Anything else before I merge this?

**Comment by maclover7** [comment]
Why did you add in RSpec if there are no tests?

**Comment by dwradcliffe** [comment]
I plan on adding tests later, and this allows us to deploy with shipit. (Our version of shipit requires a green CI before you can deploy.)

**Comment by maclover7** [comment]
Gotcha. Looks ready to roll on my end. Should I merge, or will you? :ship:

**Comment by maclover7** [comment]
:+1:

**Comment by maclover7** [comment]
:+1:

**Pull request merged**